examples/AspNetCore/OData/ODataOpenApiExample/Program.cs

```diff
@@ -80,15 +80,17 @@
 
 var app = builder.Build();
 
-// Configure the HTTP request pipeline.
+// Configure HTTP request pipeline.
 
 if ( app.Environment.IsDevelopment() )
 {
-    // navigate to ~/$odata to determine whether any endpoints did not match an odata route template
+    // Access ~/$odata to identify OData endpoints that failed to match a route template.
     app.UseODataRouteDebug();
 }
 
 app.UseSwagger();
+if ( app.Environment.IsDevelopment() )
+{
```

Comment:
@commonsensesoftware should I move `app.UseSwagger();` up so I only need one 
`if ( app.Environment.IsDevelopment() )`
block

Reply:
Less blocks would be good so I don't have a problem with that. Is the established practice to disable OpenAPI docs and the Swagger UI outside of `Debug`? It's more for my edification.

Reply:
Where'd we land on this? Leave it on the outside or move it inside so it's only for development? I'm not sure the _practice_ here. Everything else looks good. If you're happy with this, then I'll merge it. 😉

Reply:
>  Is the established practice to disable OpenAPI docs and the Swagger UI outside of Debug? It's more for my edification.

General best practice is to disable UIs that might be susceptible to a web security issues (leaked creds, XSS, etc.) outside of development. The generated JSON is fine to enable in production and, in fact, might be necessary for certain integration scenarios (e.g. client generators, OpenAI plugins, API management front-ends).